picard/config.py

```diff
@@ -479,3 +491,14 @@ def get_config():
     Config objects for threads are created on demand and cached for later use.
     """
     return config
+
+
+def load_new_config(filename=None):
+    ini_file = get_config().fileName()
```

Comment:
We should use a format-agnostic variable name instead like `config_file`.
Configuration files format may change over the time.